src/main.rs

```diff
@@ -58,10 +58,35 @@ fn main() {
         },
         Commands::Screen(command) => match command {
             ScreenCommands::List => {
-                warn!("List: to be implemented");
+                let screen_command = commands::ScreenCommand::new(authentication);
+                match screen_command.list() {
+                    Ok(v) => {
+                        println!(
+                            "{}",
+                            serde_json::to_string_pretty(&v).unwrap_or_else(|_| "{}".to_string())
+                        );
+                    }
+                    Err(e) => {
+                        println!("Error occurred: {:?}", e);
+                        std::process::exit(1);
+                    }
+                }
             }
-            ScreenCommands::Get { id: _ } => {
-                warn!("Get: to be implemented");
+            ScreenCommands::Get { id } => {
+                let screen_command = commands::ScreenCommand::new(authentication);
+                match screen_command.get(id) {
+                    Ok(v) => {
+                        println!(
+                            "{}",
+                            serde_json::to_string_pretty(&v).unwrap_or_else(|_| "{}".to_string())
+                        );
+                        std::process::exit(0);
+                    }
+                    Err(e) => {
+                        println!("Error occurred: {:?}", e);
```

Comment:
stderr

You still can output result to stdout if you represent the result as json.